test/Dat.js

```diff
@@ -8,7 +8,7 @@ var fs = require("fs")
 
 describe('Dat', () => {
     describe('Share', () => {
-        it('should resolve provided path correcty', function(done) {
+        it('Resolves a provided path correctly', function(done) {
```

Comment:
Pop spaces before and after () after function

Reply:
done